includes/models/model.llms.user.certificate.php

```diff
@@ -114,6 +125,100 @@ public function can_user_view( $user_id = null ) {
 
 	}
 
+	/**
+	 * Retrieve information about the certificate background image.
+	 *
+	 * @since [version]
+	 *
+	 * @return array {
+	 *     Returns an associative array of information about the background image.
+	 *
+	 *     @type string $src        The image source url.
+	 *     @type int    $width      The image display width, in pixels.
+	 *     @type int    $height     The image display height, in pixels.
+	 *     @type bool   $is_default Whether or not the default image was returned.
+	 * }
+	 */
+	public function get_background_image() {
```

Comment:
Ohhh just noticed this method is not backward compatible.
Meaning that if someone earned a certificate with the current stable and that certificate template has a custom background image, when viewing the earned certificate with this PR I see the default background.